ACPs/84-table-preamble/README.md

```diff
@@ -21,15 +21,14 @@ The current ACPs `Preamble` is (i) not very readable and (ii) not user-friendly
 
 The following Markdown table format is proposed:
 
-| ACP                            | PR Number                                                                                    |
-| :----------------------------- | :------------------------------------------------------------------------------------------- |
-| **Title**                      | ACP title                                                                                    |
-| **Author(s)**                  | A list of the author's name(s) and optionally contact info: FirstName LastName <foo@bar.com> |
-| **Discussions-To**             | [Hyperlinked GitHub Discussion name](./README.md)                                            |
-| **Status**                     | Proposed, Implementable, Activated, Stale                                                    |
-| **Track**                      | Standards, Best Practices, Meta, Subnet                                                      |
-| **Replaces (\*optional)**      | [ACP-XX](./README.md)                                                                        |
-| **Superseded-By (\*optional)** | [ACP-XX](./README.md)                                                                        |
+| ACP                            | PR Number                                                                                                                                        |
+| :----------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Title**                      | ACP title                                                                                                                                        |
+| **Author(s)**                  | A list of the author's name(s) and optionally contact info: FirstName LastName ([@GitHubUsername](./README.md) or [email@addre.ss](./README.md)) |
```

Comment:
is `addre.ss` a typo or intended? Unclear to me

Reply:
It isn't a typo, just a placeholder hyperlink haha

Open to suggestions

Reply:
`team.rocket@example.com`